autoload/tagbar.vim

```diff
@@ -534,6 +567,20 @@ endfunction
 
 " s:CheckFTCtags() {{{2
 function! s:CheckFTCtags(bin, ftype) abort
+    " dart_ctags isn't installed as a bin
+    " must use pub command to run
+    if a:bin ==? 'dart_ctags'
```

Comment:
Will this ever conceivably be lower case? If its not likely lets just assume that:

```suggestion
    if a:bin ==# 'dart_ctags'
```